experiments/regression/uci_datasets/readme.md

```diff
@@ -1,17 +1,33 @@
 # UCI Regression - Benchmark
 
+
+| Dataset | Number of Instances | Number of Features |
+| --- | --- | --- |
+| Boston Housing | 506 | 13 |
+| Concrete Compression Strength | 1030 | 8 |
+| Energy Efficiency | 768 | 8 |
+| Kin8nm | 8192 | 8 |
+| Naval Propulsion | 11,934 | 16 |
+| Combined Cycle Power Plant | 9568 | 4 |
+| Protein Structure | 45730 | 9 |
+| Wine Quality (Red) | 1599 | 11 |
+| Yacht Hydrodynamics | 308 | 6 |
+
+
+> [!WARNING]
+> Some datasets require installing additional packages.
```

Comment:
We could list them here, or tell to download optional dependencies with pip install torch_uncertainty[xxx]